### PR TITLE
Use polyglot-release to make releases

### PR DESCRIPTION
**Pull request by mpkorstanje**
Following https://github.com/cucumber/polyglot-release/issues/11 [polyglot-release](https://github.com/cucumber/polyglot-release) automates the release process, removing the need for a long list of manual actions.

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -46,50 +61,27 @@ Anyone with permission to push to the `main` branch can prepare a release.
...
-      ```
-      changelog release $next_release --tag-format "v%s" -o CHANGELOG.md
-      ```
-    * If you don't have the `changelog` tool installed:
-        * Under `[Unreleased]` at the top, add a new `[${version}] - ${YYYY-mm-dd}` header
-        * Add a new `[${version}]` link at the bottom
-        * Update the `[Unreleased]` link at the bottom
-1. Update the version number in the relevant package decriptor(s), such as:
-    * `package.json` - just run `npm version $next_release --no-git-tag-version`, will also update `package-lock.json`
-    * `go.mod`
```
Comment (mpkorstanje) [inline RELEASING.md:67]: Go is not yet supported: https://github.com/cucumber/polyglot-release/issues/14

File: RELEASING.md [hunk 2]
```diff
@@ -46,50 +61,27 @@ Anyone with permission to push to the `main` branch can prepare a release.
...
-    * `pom.xml` **Remove the -SNAPSHOT suffix**
-    * `VERSION` (for Ruby libraries)
-    * `*.csproj` - update `<VersionNumber>`
```
Comment (mpkorstanje) [inline RELEASING.md:70]: .Net is not yet supported https://github.com/cucumber/polyglot-release/issues/15

**Comment by aslakhellesoy** [comment]
I just tested this on `@cucumber/language-server` (monoglot) and it worked right out of the box!

You've done a fantastic job with both the shell scipting and the documentation! 👏🏻👏🏻👏🏻 

The only minor flaw I could find was a few warnings that were unnecessary noise:
```
find: ./*.sln: No such file or directory
find: ./*.gemspec: No such file or directory
```
Easy to turn off.

I say we take it for a spin in a polyglot repo, and if it works fine let's ship this.

This is going to make releases soooo much easier!

File: RELEASING.md [hunk 3]
```diff
@@ -10,9 +10,24 @@ There are three parts to making a release:
...
+ * [polyglot-release](https://github.com/cucumber/polyglot-release)
+ * [changelog](https://github.com/cucumber/changelog/)
+ * Java
+   * Maven - version > 3.6
+   * JDK - version >= 11
+ * Javascript
+   * Node - version >= v14
+   * NPM - version >= 8
+ * Ruby
+   * None
```
Comment (mpkorstanje) [inline RELEASING.md:26]: This list is missing xmlstarlet for .NET

**Comment by mpkorstanje** [comment]
@mattwynne of those projects only https://github.com/cucumber/cucumber-jvm-scala isn't supported.

I think we can make a separate issue for that in polyglot and then merge these instructions.

**Comment by mpkorstanje** [comment]
Updated the wording to ensure the release docs in a project are leading and made https://github.com/cucumber/polyglot-release/issues/56 to catch what remains.

File: RELEASING.md [hunk 4]
```diff
@@ -46,51 +53,28 @@ Anyone with permission to push to the `main` branch can prepare a release.
...
+   * Look at `CHANGELOG.md` to see what has changed since the last relesase
+   * Use [semver](https://semver.org/) to decide on a version for the next release
+   * If you are bumping the `MAJOR` version of `cucumber-{jvm,js,ruby}`, see the [Major release](#major-release) section
+1. From the root of the polyglot project you are trying to release run:
+
+```
+polyglot-release <new version>
+```
 1. Wait until the `release-*` workflows in GitHub Actions have passed
 1. Rerun individual workflows if they fail
```
Comment (mattwynne) [inline RELEASING.md:71]: Could add here:

> If there's something wrong with the release workflows themselves, you can make changes and `git push --force` to the release branch to re-run the workflow. Don't forget to merge any changes back to `main` afterwards.

**Pull request merged**